hack/generate-manifest-windows.sh

```diff
@@ -44,6 +47,7 @@ function print_help {
 RUNTIME=""
 MODE="dev"
 KEEP=false
+OVSRunMode="service"
```

Comment:
nit: follow the naming style, it should be OVS_RUN_MODE.
But here I feel a bool is more proper, `INCLUDE_OVS=false`, as a specific mode value is not meaningful in this script.